www/src/pages/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import Layout from '../components/creative-layout'
+import {Parallax, ParallaxLayer} from 'react-spring/renderprops-addons'
+import styles from '../components/creative-layout.module.scss'
+
+import spellsource from '../assets/spellsource-resized.png'
+import gameEnvironment from '../assets/sector-5.png'
+import cloud from '../assets/transparent-cloud.png'
+import champions from '../assets/champions-without-names.png'
+
+const url = (name, wrap = false) => `${wrap ? 'url(' : ''}https://awv3node-homepage.surge.sh/build/assets/${name}.svg${wrap ? ')' : ''}`
+
+const Index = () => {
+  let parallax
+  return (
+    <Layout>
+      <section className={styles.fullscreenDiv}>
```

Comment:
![localhost_8000_(iPhone X)](https://user-images.githubusercontent.com/2229300/86046642-6b3bbf00-ba02-11ea-8b6c-e70af79b932a.png)

horizontal lines should go to the edge
maybe rethink how the navbar is presented, can we get it to scroll horizontally maybe? without a scrollbar, touch horizontal scroll only, with the icon
the logo is pretty tiny

![localhost_8000_(iPhone X) (1)](https://user-images.githubusercontent.com/2229300/86046763-a76f1f80-ba02-11ea-8f3c-2e7ec09e46c2.png)


the map should probably be centered here
the heroes are extra tiny
heroes are weirdly floating over the map
lots of dead area here

![localhost_8000_(iPhone X) (2)](https://user-images.githubusercontent.com/2229300/86046829-c2da2a80-ba02-11ea-992f-d8058c790d0d.png)


text in index.js probably needs a little bit of padding on the side

![localhost_8000_(iPhone X) (3)](https://user-images.githubusercontent.com/2229300/86047056-1d738680-ba03-11ea-88f7-837ef2855998.png)
add some logos for the buttons (they can just be big shiny pixel art buttons

the footer looks weird not centered right line is off